<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbilitySimulation</class>
 <widget class="QDialog" name="AbilitySimulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ability Simulation</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="partsList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="abilityLevel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Perfect</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Professional</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Good</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Average</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Awful</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="abilityLevelLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ability:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="useAbilitySimulation">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use ability simulation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyToAll">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply to all</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="highlightDifficult">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Highlight difficult areas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AbilitySimulation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AbilitySimulation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
